manager/src/main/java/com/redhat/service/smartevents/manager/api/user/ErrorsAPI.java

```diff
@@ -54,7 +54,7 @@ public class ErrorsAPI {
 
     @APIResponses(value = {
             @APIResponse(description = "Success.", responseCode = "200",
-                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = ErrorListResponse.class))),
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, ref="ErrorListResponse", implementation = ErrorListResponse.class))),
```

Comment:
Can you please verify if there is any validation of this ref during the compilation? For example when there is a typo or the schema is renamed.

We might also consider to add an enforcer rule to make sure we always use ref also in the future

Reply:
I have simplified the annotations as demonstrated here https://download.eclipse.org/microprofile/microprofile-open-api-3.0-RC4/microprofile-openapi-spec-3.0-RC4.html#_schema

see my last commit.

There is need to use just `implementatiton` without `type`. Do we still need some validation of this? @danielezonca